crates/ethereum/engine-primitives/src/payload.rs

```diff
@@ -248,6 +248,20 @@ impl PayloadBuilderAttributes for EthPayloadBuilderAttributes {
     }
 }
 
+impl Default for EthPayloadBuilderAttributes {
+    fn default() -> Self {
+        Self {
+            id: PayloadId::new([0; 8]),
+            parent: B256::default(),
+            timestamp: 0,
+            suggested_fee_recipient: Address::ZERO,
+            prev_randao: B256::default(),
+            withdrawals: Withdrawals::default(),
+            parent_beacon_block_root: None,
+        }
+    }
+}
```

Comment:
this can probably be derived

Reply:
Requires this change in `alloy` I think, PTAL
https://github.com/alloy-rs/alloy/pull/1442